Demo/DemoApp/DemoApp/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
I kept this around to make the DemoApp fully decoupled from the existing test rig (per discussion on Slack).